Fixes #3723 - MXP &text; not working as expected

gcms opened the pull request:
<!-- Keep the title short & concise so your mom can understand it -->
#### Brief overview of PR changes/additions
- Fixes problem with interpolating `&text;` in standalone `<SEND>` tags

#### Motivation for adding to Mudlet
Bugfix

#### Other info (issues closed, discussion etc)
Closes #3723 

MUDs that use this construct: Age of Elements (in standalone `<send>`) and Two Towers (in custom elements)

gcms committed:
Fixes #3723 - MXP &text; not working as expected

- Fixes problem with interpolating `&text;` in standalone `<SEND>` tags

mpconley commented:
Thanks for the quick turnaround.  You could connect to ageofelements.org 7680 to check on this.  In the first room, you could click on `button` in the description to test.  I believe there is still an issue with the semicolon being included in what is sent (i.e. see in this image it shows `push button;` instead of `push button`).
<img width="606" alt="Screen Shot 2020-05-03 at 7 43 28 AM" src="https://user-images.githubusercontent.com/3058178/80913329-d7a38600-8d11-11ea-8c2b-75287c93a8cc.png">

gcms commented:
> Thanks for the quick turnaround. You could connect to ageofelements.org 7680 to check on this. In the first room, you could click on `button` in the description to test. I believe there is still an issue with the semicolon being included in what is sent (i.e. see in this image it shows `push button;` instead of `push button`).
> <img alt="Screen Shot 2020-05-03 at 7 43 28 AM" width="606" src="https://user-images.githubusercontent.com/3058178/80913329-d7a38600-8d11-11ea-8c2b-75287c93a8cc.png">

Hrmm, I see. Similar problem of asymmetry. Semicolons are stripped from custom elements but not from standalone SEND tags.

https://github.com/Mudlet/Mudlet/blob/cd5294609d92566e8912091b0c99c14863c979be/src/TBuffer.cpp#L1498-L1503

https://github.com/Mudlet/Mudlet/blob/cd5294609d92566e8912091b0c99c14863c979be/src/TBuffer.cpp#L1672-L1675

I am having trouble to connect to this MUD to check it
![Screenshot from 2020-05-03 12-55-38](https://user-images.githubusercontent.com/4356833/80918867-77c2d480-8d3d-11ea-8719-eeff2299d270.png)

gcms committed:
Fixes problem with semicolons not being replaced

- Similar problem to the uppercased `&TEXT;`, the custom tag handler stripped the semicolons, but the handler for send tag didn't.

mpconley commented:
@gcms This passes testing on AoE!  Thank you!

vadi2 reviewed (approved):
Thanks for catching this before release! :)
